common/src/main/java/com/wynntils/utils/type/IterationDecision.java

```diff
@@ -6,5 +6,6 @@
 
 public enum IterationDecision {
     CONTINUE,
-    BREAK
+    BREAK,
```

Comment:
I'm a bit confused now. You add this here, but don't you have to use it in the styled text iterator method implementation as well?

Reply:
I should, yes.. I dont know how it worked..

Reply:
Also, the comment is not so clear. "next elements". I assume you mean "all remeaining elements", giving how this is used in the item handler. But the comments make it sound almost like you just skip the next part, and that the plural -s is just a typo.